docs/source/examples/combine_ddd_zne.md

```diff
@@ -0,0 +1,192 @@
+
+    # Add CNOT gates to entangle the first qubit with each of the other qubits
+    for i in range(1, num_qubits):
+        circuit.append(cirq.CNOT(qubits[0], qubits[i]))
+        # Set qubits to idle for specfied number of steps
+        # in the form of Identity gates
+        for step in range(idle_steps):
+            circuit.append(cirq.I(q) for q in qubits)
+
+    return circuit
+```
+
+```python
+num_qubits = 6
+circuit = ghz(num_qubits, 2)
+print(circuit)
+```
+
+## Noise model and executor
+
+**Importantly**, since DDD is designed to mitigate time-correlated (non-Markovian) noise, we simulate systematic $R_z$ rotations and depolarising noise applied to each qubit after each time step. This corresponds to noise which is strongly time-correlated and, therefore, likely to be mitigated by DDD.
+
+We use an [executor function](../guide/executors.md) to run the quantum circuit with the noise model applied.
+
+```python
+def execute(
+    circuit: cirq.Circuit, 
+    rz_noise: float = 0.01,
+    depolar_noise: float = 0.0005
+    ) -> MeasurementResult:
+    """
+    Execute a circuit with R_z dephasing noise of strength ``rz_noise`` and depolarizing noise ``depolar_noise``
+    """
+    # Simulate systematic dephasing (coherent RZ) on each qubit for each moment.
+    circuit = circuit.with_noise(cirq.rz(rz_noise))
+
+    # Simulate systematic depolarizing on each qubit for each moment.
+    circuit = circuit.with_noise(cirq.bit_flip(depolar_noise))
+
+    # Measure out all qubits
+    circuit += cirq.measure(*sorted(circuit.all_qubits()), key="m")
+
+    # Use a noise simulator
+    simulator = cirq.DensityMatrixSimulator()
+
+    # Run the circuit 1000 times
+    result = simulator.run(circuit, repetitions=1000)
+
+    # Retrieve the measurement results in bitstring form
+    bitstrings = result.measurements["m"]
+
+    return MeasurementResult(bitstrings)
+```
+
+Let's see what the execute function returns when we call it on our GHZ circuit, just leaving the default noise levels:
+
+```python
+res = execute(circuit)
+res.to_dict() # Dictionary for more convenient visualization
+```
+
+The `'counts'` key represents the number of times a particular measurement bitstring occurred. 
+
+Since the default execute function includes noise in this case, we see that most of the results are all ones or all zeros, corresponding to perfect entanglement between all qubits.
+
+A few others however are one or more bitflips away from either of these states. It is these types of errors we want to mitigate. 
+
+
+## Observable
+
+In this example, we just want to check if we have achieved entanglement across all qubits. In this case, we will measure the observable $⨂_{i=1}^n​X_i$ or measuring `X` on all qubits. 
+
+This corresponds to projecting the state onto either the $∣+⟩^{⊗n}$ or $∣−⟩^{⊗n}$ basis. For a perfect GHZ state, this observable will have an expectation value of 1, which corresponds to all qubits being in the same state.
```

Comment:
LaTex syntax should be working here, is it not?

Reply:
It is, it was just quicker for me to copy/paste the unicode versions of these symbols. Happy to switch back to latex if there's an issue.